src/custom/state/claim/hooks/index.ts

```diff
@@ -288,17 +322,26 @@ function useDeploymentTimestamp(): number | null {
   const { chainId } = useActiveWeb3React()
   const vCowContract = useVCowContract()
   const [timestamp, setTimestamp] = useState<number | null>(null)
+  const isMounted = useIsMounted()
 
   useEffect(() => {
     if (!chainId || !vCowContract) {
       return
     }
 
-    vCowContract.deploymentTimestamp().then((ts) => {
-      console.log(`Deployment timestamp in seconds: ${ts.toString()}`)
-      setTimestamp(ts.mul('1000').toNumber())
-    })
-  }, [chainId, vCowContract])
+    fetchDeploymentTimestamp(vCowContract)
+      .then((timestamp) => {
+        if (isMounted.current) {
+          setTimestamp(timestamp)
+        }
+      })
+      .catch(() => {
+        if (isMounted.current) {
+          setTimestamp(null)
+          console.log('vCowContract Deployment Timestamp fetch failed')
```

Comment:
Shouldn't we print this using `console.error` Also, shouldn't we include the error object so we see the stack trace?